src/app/views/user/user-settings/user-settings.component.scss

```diff
@@ -39,68 +38,69 @@
         }
 
         .file {
-          width: 85px;
+          cursor: pointer;
           height: 85px;
           opacity: 0;
-          cursor: pointer;
+          width: 85px;
 
           &[disabled] {
             pointer-events: none;
           }
         }
 
         .icon-camera {
-          width: 85px;
+          bottom: 0;
```

Comment:
Properties should be ordered position, top, right, bottom, left, display, width, height, pointer-events

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)